docs/geedocs/5.3.2/raw_sources/answer/releaseNotes/relNotesGEE5_3_2.rst.txt

```diff
@@ -35,7 +40,7 @@ Release notes: Open GEE 5.3.2
       -  Red Hat Enterprise Linux version 6.x and 7.x, including the
          most recent security patches
       -  CentOS 6.x and 7.x
-      -  Ubuntu 14.04 LTS and 16.04 LTS
+      -  Ubuntu 16.04 LTS

```

Comment:
Maybe note somewhere that support for Ubuntu 14.04 has been removed since this is the first release to do that.

Reply:
> Maybe note somewhere that support for Ubuntu 14.04 has been removed since this is the first release to do that.

Done